vulkaninfo/vulkaninfo.h

```diff
@@ -1297,6 +1296,13 @@ struct AppGpu {
                 VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG,
                 VK_FORMAT_PVRTC2_4BPP_SRGB_BLOCK_IMG,
             },
+            {
```

Comment:
Is this stuff RANGE-related?  If not, it should be in a separate commit.

Reply:
It is related, but I will make it a separate commit for clarity